ci/index.ts

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env node
+import 'source-map-support/register';
+import * as cdk from '@aws-cdk/core';
+import * as codebuild from '@aws-cdk/aws-codebuild';
+import * as logs from '@aws-cdk/aws-logs';
+import { Bucket } from '@aws-cdk/aws-s3';
+import { PolicyStatement } from '@aws-cdk/aws-iam';
+// Utils
+import { valueFromContext } from '../lib/utils/context-utils';
+
+const app = new cdk.App();
+
+export class CiStack extends cdk.Stack {
+  constructor(scope: cdk.Construct, id: string, props?: cdk.StackProps) {
+    super(scope, id, props);
+
+    // Parameters
+    const ghOwner = new cdk.CfnParameter(this, 'GitHubOwner', {
+      type: "String",
+      description: "The GitHub organization you own.",
+      default: "aws-quickstart"
+    });
+
+    const contextLocation = new cdk.CfnParameter(this, 'ContextLocation', {
```

Comment:
I like it, better than dealing with any other secrets in github. The CI account will have the S3 bucket to which we should have access and it will decouple us from the quickstart team.

Reply:
Parameters don't quite work here because I need to resolve the string value for updating CodeBuild service role policy prior to synthesis. Going to update the PR to read from context instead.